### DISCUSSION TITLE
An error occurs when I use mvtlayer offline

### DISCUSSION THREAD

**Discussion by zzkzzk1984** [visgl/deck.gl#7516] [Q&A] [answered]
An error occurs when I use mvtlayer offline. How can I solve it?

DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://unpkg.com/@loaders.gl/mvt@3.2.12/dist/mvt-worker.js' failed to load.

**Comment by kylebarron** [comment, marked as answer]
You can set the worker url to a url on localhost: https://deck.gl/docs/developer-guide/loading-data#loaders-and-web-workers

**Reply by zzkzzk1984** [reply]
thank you
